expression/aggregation.go

```diff
@@ -332,6 +367,11 @@ func (sf *sumFunction) GetGroupResult(groupKey []byte) (d types.Datum) {
 	return sf.getContext(groupKey).Value
 }
 
+// GetPartialResult implements AggregationFunction interface.
+func (sf *sumFunction) GetPartialResult(groupKey []byte) []types.Datum {
+	return []types.Datum{sf.getContext(groupKey).Value}
```

Comment:
Why not call GetGroupResult and convert its result to []Datum? So as the following functions.